tests/templates.test.js

```diff
@@ -76,18 +77,18 @@ describe('Templates class', () => {
 
     it('only loads files once', async () => {
         const EJS = require('ejs');
-        const mockLoader = jest.fn(filename => `<%= "${filename}" -%>`);
+        const mockLoader = jest.fn(filename => `<%= ${JSON.stringify(filename)} -%>`);
```

Comment:
Without this `\` in the file path would be treated as part of an escape sequence (e.g.: `C:\nodejs\...\kumascript\tests\fixtures\templates\test.ejs` would be interpreted as:
<pre><code>
C:
odejs...kumascript	ests&#x0C;ixtures	emplates	est.ejs
</code></pre>

Reply:
Same here. That's a very useful comment. Let's play Jeopardy, the comment should be the answer to "Why does the file name need to be JSON.stringify escaped on this line???"